examples/tests/valid/resource.w

```diff
@@ -7,6 +7,7 @@ class Foo {
 
   init() {
     this.c = new cloud.Counter();
+    this.inflightField = 1;
```

Comment:
Remove this line (`inflightField` is initialized inside `inflight init()` already)